app/services/vendor_validator_result_storage.rb

```diff
@@ -0,0 +1,24 @@
+class VendorValidatorResultStorage
+  TTL = Figaro.env.session_timeout_in_minutes.to_i.minutes.seconds.to_i
```

Comment:
Does this TTL get updated or only set once when first called? I'm thinking about the scenario where the user starts proofing at 12:00, this TTL is then set to expire at 12:08, but the user takes longer than that to finish proofing. For example, after they submit the first profile page, let's say the vendor takes a minute to respond. Then, the user stays idle for 7 minutes on the financial info page because they had to go find their credit card. It's now 12:08, but the session is still active because their last request was only 7 minutes ago. So then they enter their credit card, but now the Redis TTL has expired. Will this cause any issues?